python/pfs/utils/fiberids.py

```diff
@@ -3,6 +3,8 @@
 import os
 
 import eups
+from pfs.utils.constants import MISSING_VALUE_UINT16
```

Comment:
Please don't use this constant for this purpose.
Firstly, a globally defined constant (such as from `pfs.utils.constants`) isn't always appropriate to a particular class -- so the constant should be defined inside the class, not externally, so the association is clear.
Secondly, the use of Hungarian notation requires users to know implementation details of the class.